amazon_transcribe/serialize.py

```diff
@@ -130,6 +134,36 @@ def serialize_start_stream_transcription_request(
             )
         )
 
+        headers.update(
+            self._serialize_bool_header(
+                "identify-language",
+                request_shape.identify_language,
+            )
+        )
+
+        headers.update(
+            self._serialize_str_header(
+                "preferred_language",
```

Comment:
```suggestion
                "preferred-language",
```
I think it's a typo.